command.go

```diff
@@ -787,8 +787,8 @@ func (c *Command) initHelpCmd() {
 				}
 			},
 		}
+		c.AddCommand(c.helpCommand)
```

Comment:
If I set a custom help command via `SetHelpCommand()` then `helpCommand` would be non nil but `AddCommand` would never have been called. Right?